Simplenote/src/main/res/values/strings.xml

```diff
@@ -216,7 +216,8 @@
     <string name="wordpress_post_draft">Set status to draft</string>
     <string name="uploading_post">Uploading post</string>
     <string name="copy_url">Copy URL</string>
-    <string name="success">Successfully posted to</string>
+    <string name="copied_to_clipboard">Copied to clipboard</string>
```

Comment:
We could probably get away with using this string instead of adding a new one?

https://github.com/Automattic/simplenote-android/blob/f888406e957b96c44b6b19b4117b40cb480b4b7b/Simplenote/src/main/res/values/strings.xml#L101

Reply:
I was going to use `link_copied` originally, but I created `copied_to_clipboard` so that users wouldn't get confused by the "URL" term on the button and the "Link" term in the message.  I agree; one string is better than two.  What about removing `copied_to_clipboard`, using `link_copied` for the confirmation message, removing `copy_url`, and using `copy` for the button?  That way the buttons will just say "Share" and "Copy" with the "Link copied to clipboard" confirmation message.

Reply:
> What about removing `copied_to_clipboard`, using `link_copied` for the confirmation message, removing `copy_url`, and using `copy` for the button? That way the buttons will just say "Share" and "Copy" with the "Link copied to clipboard" confirmation message.

Sounds good 👍

Reply:
I updated those references in 77b9ee1 and f4361d4.